lib/widgets/rules_buttons.dart

```diff
@@ -1,20 +1,24 @@
 import 'package:flutter/material.dart';
 import 'package:tic_tac_toe/widgets/dropdown.dart';
 
-import 'game_view.dart';
-
-String selectedValue = '3';
-String winValue = 'Win';
-int inRowValue = 3;
-
 class RulesChangingButtons extends StatefulWidget {
-  const RulesChangingButtons({super.key});
+  final void Function(int inRowValue, int whoWins) onRulesChanged;
+
+  const RulesChangingButtons({
+    Key? key,
+    required this.onRulesChanged,
+  }) : super(key: key);
 
   @override
   RulesChangingButtonsState createState() => RulesChangingButtonsState();
 }
 
 class RulesChangingButtonsState extends State<RulesChangingButtons> {
+  String selectedValue = '3';
+  String winValue = 'Win';
+  int inRowValue = 3;
+  int whoWins = 0;
```

Comment:
Replace whoWins to boolean with name 'rowBuilderWins', to make it more understandable.

Reply:
Done